components/ItemCripto/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import {Image, Text, View} from 'react-native';
+import {ItemCriptoProps} from '../../utils/interfaces';
+import {theme} from '../../utils/theme';
+
+import {
+  CriptoContainer,
+  SeconContainer,
+  CriptoName,
+  CriptoPercentage,
+  ImageArrow,
+} from './styles';
+
+const ItemCripto = ({item}: ItemCriptoProps) => (
+  <CriptoContainer>
+    <SeconContainer>
+      <Image source={item.img} />
+      <View>
+        <CriptoName>{item.name}</CriptoName>
+        <Text>{item.symbol}</Text>
+      </View>
+    </SeconContainer>
+    <View>
+      <CriptoName>{item.price}</CriptoName>
+      {item.percentage > 0 ? (
```

Comment:
You are repeating code.
CriptoPercentage should take a boolean in it's props, that indicates wich color use, and ImageArrow should have a validation in the source to see if it have to use the red or the green arrow. In this way you will end only using one componen block

Reply:
I release the changes in commit: ff88652